huxley/www/static/js/huxley/components/CountrySelect.js

```diff
@@ -34,14 +33,22 @@ var CountrySelect = React.createClass({
 
   renderCommitteeOptions: function() {
     return this.props.countries.map(function(country) {
-      if (!country.special) {
+      var index = this.props.countryPreferences.indexOf(""+country.id)
+      if(index < 0){
         return (
           <option key={country.id} value={country.id}>
             {country.name}
           </option>
         );
+
+      }else{
+        return (
```

Comment:
And while we're here, small style nits, add spaces around keywords and binary operators:
```
var foo = bar.indexOf("" + country.id)
if (...) {
  ...
} else {
  ...
}
```
No need to update a diff just for style nits, but since there are substantive changes to be made, add these in there too :)